pkgs/by-name/he/heliocron/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "heliocron";
+  version = "0.8.1";
+
+  src = fetchFromGitHub {
+    owner = "mfreeborn";
+    repo = pname;
```

Comment:
```suggestion
    repo = "heliocron";
```

Reply:
Is there documentation stating that a hardcoded string is preffered to pname? I tried to read through all of the contributing documentation and didn't see any, so I'd love to know if there's something else I've missed!

Reply:
pname is a bit special, here is a tracking issue https://github.com/NixOS/nixpkgs/issues/277994